solarkraft/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+// A dev container for working with Soroban smart contracts.
```

Comment:
The descriptions of the both containers is identical. What's the difference?

Reply:
Aaah, I thought I did changed it 😄 
[e171c2a](https://github.com/freespek/solarkraft/pull/31/commits/e171c2a477c78d06a0b0d3c87519767871d6f56a)